src/_P031_SHT1X.ino

```diff
@@ -69,8 +69,22 @@ boolean Plugin_031(byte function, struct EventStruct *event, String& string)
         Plugin_031_DATA_Pin = CONFIG_PIN1;
         Plugin_031_CLOCK_Pin = CONFIG_PIN2;
         if (Settings.TaskDevicePin1PullUp[event->TaskIndex]) {
+          #if defined(ESP8266)
+          String log = F("SHT1X: Setting ");
+          if (Plugin_031_DATA_Pin == 16)
+          {
+            input_mode = INPUT_PULLDOWN_16;
+            log += F("PullDown on pin ");
+          }
+          else
+          {
+            input_mode = INPUT_PULLUP;
+            log += F("PullUp on pin ");
+          }
+          #else
```

Comment:
This will change in another commit that's ready to be merged regarding this plugin.

Reply:
OK, I have removed this change.